build/yamls/flow-visibility-e2e.yml

```diff
@@ -319,7 +319,7 @@ spec:
         spec:
           containers:
             - name: clickhouse
-              image: projects.registry.vmware.com/antrea/theia-clickhouse-server:21.11
+              image: projects.registry.vmware.com/antrea/clickhouse-server:23.4
```

Comment:
This file is originally designed as auto-generated by a script in Theia, as stated in the first several lines in this file.
But as we are removing the dependency on Theia, the previous strategy we use to auto-generate the e2e yaml, which will use the theia-clickhouse-server image, may not work. Besides, the changes at theia side may introduce unnecessary complexity to this file. Maybe we need to think about removing the first several lines in this file or rewrite the e2e yaml generation script at theia side.

Reply:
We discussed offline - we will keep `flow-visibility-e2e.yml` as it is for now. Considering future data schema changes, it is better to use a script to auto-generate this file, so that we can make sure the data schema in two repositories in-sync. 

Later we will need to update the [generate-manifest.sh](https://github.com/antrea-io/theia/blob/main/hack/generate-manifest.sh) in theia. That script is a bit of out-dated, when generating `flow-visibility-e2e.yml` we would like to remove new features like data schema management, only keep necessary resources. I will open an issue to keep track of it.